src/__tests__/old-act.js

```diff
@@ -4,32 +4,92 @@ jest.mock('../react-dom-16.9.0-is-released', () => ({
   reactDomSixteenPointNineIsReleased: true,
 }))
 
-jest.mock('react-dom/test-utils', () => ({
-  act: cb => {
-    const promise = cb()
-    return {
-      then() {
-        console.error('blah, do not do this')
-        return promise
-      },
-    }
-  },
-}))
-
 test('async act works even when the act is an old one', async () => {
+  jest.mock('react-dom/test-utils', () => ({
```

Comment:
I really prefer to not put `jest.mock` inside a test. It makes people think that the mock only applies for that test and not the others which is not correct. Could we leave this at the top level as before?